core/templates/pages/exploration-player-page/templates/lesson-information-card-modal.component.html

```diff
@@ -26,12 +26,14 @@ <h4 [ngClass]="(checkpointCount === 1) ? 'd-none' : 'progress-bar-label-text'">
           Learning Progress
         </h4>
         <div class="progress-bar-container"
-             [ngClass]="{'d-none': checkpointCount === 1, 'multiple-checkpoints-bar': checkpointCount > 1, 'reverse-checkpoint-order': isLanguageRTL()}">
+             [ngClass]="{'d-none': checkpointCount === 1, 'multiple-checkpoints-bar': checkpointCount > 1, 'reverse-checkpoint-order': isLanguageRTL()}" 
+             aria-hidden="true">
           <div class="lesson-progress-bar"
                [ngClass]="{'reverse-checkpoint-order': isLanguageRTL()}">
             <div class="completed-progress-bar"
                  [style.width.%]="getCompletedProgressBarWidth()">
             </div>
+            <span class="progress" attr.aria-label="'Completed progress : ' + getCompletedProgressBarWidth() + '%'"></span>
```

Comment:
Hm, won't the screenreader still read the "{{ idx + 1 }}" part below? Please check the expected behaviour from the issue description.